### DISCUSSION TITLE
The Gate never gets triggered

### DISCUSSION THREAD

**Discussion by atharhashmi** [opcodesio/log-viewer#274] [Q&A] [answered]
@arukompas hello.
This may be something real simple but I am out of ideas as to what's going on. I have defined the `downloadLogFile`, `deleteLogFile`, and `viewLogViewer` gates in AuthServiceProvider but none of these gates every get triggered, what am I doing wrong? 
One thing to keep in mind is that I am not using Sanctum and that could be the delta between what I am expecting and what's actually happening.

Other than that, great package!

Thanks.

**Comment by arukompas** [comment, marked as answer]
Hey @atharhashmi , is the Authorization middleware used in both `'middleware'` and `'api_middleware'` properties in `config/log-viewer.php`? Share your log viewer config if not sure, or reset it to default (delete and re-publish)

**Reply by atharhashmi** [reply]
How silly of me, applying the `auth` middleware was the solution. Thanks @arukompas , this is a great package.
